splice_access_api/src/main/java/com/splicemachine/access/configuration/HBaseConfiguration.java

```diff
@@ -108,7 +108,7 @@
      * Defaults to 8192
      */
     public static final String TIMESTAMP_BLOCK_SIZE = "splice.timestamp_server.blocksize";
-    protected static final int DEFAULT_TIMESTAMP_BLOCK_SIZE = 8192;
+    protected static final int DEFAULT_TIMESTAMP_BLOCK_SIZE = 32768;
```

Comment:
Updated comment. We increase the block size in accordance with the increase in the timestamp step increment.